frontend/src/services/configuraion-service.js

```diff
@@ -0,0 +1,40 @@
+export const configurationService = {
+    getDisplayableColumns
+};
+
+
+function getDisplayableColumns(type) {
+    if (type === "exp") {
+        return JSON.parse(localStorage.getItem('exp-display-columns')) ||
```

Comment:
Question: what is localStorage doing here? I looked for code that was writing to localStorage but didn't find it. I'm wondering if this is needed any longer.

Reply:
Making the addition/removal of columns configurable based on user preferences is a requirement for the SMILES project. Therefore, I have started using local storage for this feature. 
The remaining part of the functionality will be implemented within this project.